script.js

```diff
@@ -0,0 +1,34 @@
+$(document).ready(function(){
+
+
+  $(".target").click(function () {
+
+  if($(this).hasClass('clickable')){
+
+      $(this).addClass("light");
+      unClickableAll();
+      console.log('clicked');
+
+      setTimeout(function () {
+        $(".target").removeClass("light");
+        clickableAll();
+      }, 3000);
+
```

Comment:
This is great, specially the fact that the responsibility for the functions has been separated